include/tvm/runtime/module.h

```diff
@@ -111,7 +111,7 @@ class Module : public ObjectRef {
  *
  * \endcode
  */
-class ModuleNode : public Object {
+class TVM_DLL ModuleNode : public Object {
```

Comment:
@tqchen We added `TVM_DLL` so that the ExampleJsonModule can inherit it in the test. It looks that this is also used by other classes like `DeviceAPI` i.e.. Is this the correct change? otherwise, please advice.

Reply:
sgtm